crates/alerter/Cargo.toml

```diff
@@ -12,7 +12,7 @@ reqwest = { version = "0.11", features = ["json"] }
 serde = { version = "1.0", features = ["derive"] }
 shared = { path = "../shared" }
 clap = { version = "3.1", features = ["derive", "env"] }
-tokio = { version = "1.15", features = ["macros", "time"] }
+tokio = { version = "1.15", features = ["macros", "time", "rt-multi-thread"] }
```

Comment:
Unrelated change, but I needed it when building the `alerter` crate individually, otherwise:
```
% cargo build -p alerter
...
   Compiling alerter v0.1.0 (/Users/nlordell/Developer/cowprotocol/services/crates/alerter)
error: The default runtime flavor is `multi_thread`, but the `rt-multi-thread` feature is disabled.
   --> crates/alerter/src/main.rs:277:1
    |
277 | #[tokio::main]
    | ^^^^^^^^^^^^^^
    |
    = note: this error originates in the attribute macro `tokio::main` (in Nightly builds, run with -Z macro-backtrace for more info)

error: could not compile `alerter` due to previous error
```